Kudu.Services.Web/DebugConsole/Default.cshtml

```diff
@@ -1,186 +1,186 @@
-﻿@{

```

Comment:
I don't think the PR actually changes any thing in that file other than whitespace. It's hard to see on github, which is why I was trying to use a better diffing tool to look at the PR :)